opal/include/opal/sys/arm64/atomic.h

```diff
@@ -188,7 +188,7 @@ static inline bool opal_atomic_compare_exchange_strong_rel_64(opal_atomic_int64_
     int tmp;
     bool ret;
 
-    __asm__ __volatile__("1:  ldxr    %0, [%2]        \n"
+    __asm__ __volatile__("1:  ldaxr   %0, [%2]        \n"
```

Comment:
Unnecessary acquire.

Reply:
Yes, you are right.  strong_rel no need to change, only  exchange_strong_ need change to stlxr.  I updated